### PR TITLE
feat: improve logging messages

### PR DESCRIPTION
**Pull request by radohanculak**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Comment by radohanculak** [comment]
[DRFT-1320](https://issues.redhat.com/browse/DRFT-1320)

**Review by msager27 — COMMENTED** [review]
As a general comment, maybe it would make sense to have the logger function retrieve the account number and prepend each message with it?  This way, in addition to not having to change every message, if we ever want to change the message format (and/or replace with org id), it's in one place.

File: system_baseline/global_helpers.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 @global_helpers_bp.before_app_request
 def log_username():
     view_helpers.log_username(current_app.logger, request)
-    message = "logged username"
+    account_number = view_helpers.get_account_number(request)
```
Comment (petrblaho) [inline system_baseline/global_helpers.py:13]: Tests are failing here with this `GET /mgmt/v0/status HTTP/1.1` request. This one is used by probes in openshift and thus there is no headers to get the account number from.

Error is as following:
```
[2023-03-30 10:49:51 +0000] [14] [ERROR] Exception on /mgmt/v0/status [GET]
Traceback (most recent call last):
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/app.py", line 1948, in full_dispatch_request
    rv = self.preprocess_request()
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/lib/python3.9/site-packages/flask/app.py", line 2242, in preprocess_request
    rv = func()
  File "/opt/app-root/src/system_baseline/global_helpers.py", line 13, in log_username
    account_number = view_helpers.get_account_number(request)
  File "/opt/app-root/.pypoetry/cache/virtualenvs/system-baseline-backend-cIokWeWu-py3.9/src/kerlescan/kerlescan/view_helpers.py", line 23, in get_account_number
    identity = json.loads(base64.b64decode(auth_key))["identity"]
  File "/usr/lib64/python3.9/base64.py", line 80, in b64decode
    s = _bytes_from_decode_data(s)
  File "/usr/lib64/python3.9/base64.py", line 45, in _bytes_from_decode_data
    raise TypeError("argument should be a bytes-like object or ASCII "
TypeError: argument should be a bytes-like object or ASCII string, not 'NoneType'
```

**Review by petrblaho — CHANGES_REQUESTED** [review]

**Comment by radohanculak** [comment]
> As a general comment, maybe it would make sense to have the logger function retrieve the account number and prepend each message with it? This way, in addition to not having to change every message, if we ever want to change the message format (and/or replace with org id), it's in one place.

Good idea, thank you for suggestion. I've just created a PR in kerlescan with this improvement, so I am closing this PR.

**Pull request closed without merging**